src/Leaflet.VectorGrid.js

```diff
@@ -60,6 +63,21 @@ L.VectorGrid = L.GridLayer.extend({
 						}
 					}
 
+					// The user asked to be notified if point type features added to the current vt layer
+					if (pointFeatureHook && feat.type === 1) {
```

Comment:
Features other than points can be split into multiple tiles no ? So the callback will be called multiple times for the same actual object on the map right ? It seemed to me that it would not be easy to manage for the user. Points are straightforward. Butl I agree, I would love to make it more generic.

It did not feel right to pass the created layer or the vt feature as they are not standard objects and their geometries are relative to the tile they are in. I did not want to let the user make the calculations to transform coords in the tiles into coords in the map. Also if the user chooses to configure empty style on a vt layer and create a marker manually instead of letting VectorGrid render a pseudo marker, then there is no layer to be created, rendered and sent back to the user by VectorGrid.